src/stacks-hierarchy/Root_TabNavigatorStack/TabNav_TicketingStack/Ticketing_TicketTabNavStack/TicketTabNav_PurchaseTabScreen/TicketTabNav_PurchaseTabScreen.tsx

```diff
@@ -46,13 +52,35 @@ export const TicketTabNav_PurchaseTabScreen = ({navigation}: Props) => {
       navigation.navigate('Root_LoginRequiredForFareProductScreen', {
         fareProductTypeConfig,
         afterLogin: {
-          screen: 'Root_PurchaseOverviewScreen',
+          screen: 'Root_ActiveTokenOnPhoneRequiredForFareProductScreen',
```

Comment:
Shouldn't there be a check here on what token the user currently has? We only want to show this screen when  travel card is the currently activated token.